apps/etterlatte-beregning/src/main/kotlin/avkorting/AvkortingService.kt

```diff
@@ -133,6 +134,21 @@ class AvkortingService(
         }
     }
 
+    fun hentHarInntektNesteAar(behandlingId: UUID): Boolean {
```

Comment:
Dette vil vel fungere men jeg tror det er bedre å helder sende sakid og et eksplisitt år vi sjekker i mot.
Også trenger man ikke finne selve inntekten/grunnlaget. Det holder å se at det finnes et årsoppgjør for det året vi sender inn som parameter.

Reply:
Året kan utledes fra fomDato på packet